src/components/modals/ConnectWalletModal/styles.css

```diff
@@ -0,0 +1,84 @@
+.modalContainer {
+  position: fixed;
+  display: flex;
+  background-color: var(--modal-bg);
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+}
+.buttonLink {
+  border: none;
+  font-family: 'Rubik';
+  font-size: 16px;
+  font-weight: 500;
+  text-decoration: underline;
+  color: var(--text-white);
+  background: none;
+  cursor: pointer;
+}
+.modalCloseButton {
+  display: flex;
+  position: absolute;
+  top: 10px;
+  right: 10px;
+  color: var(--text-white);
+  padding: 3px;
+  width: fit-content;
+  height: fit-content;
+  border: 2px solid var(--bg-color);
+  background: none;
+  border-radius: 25px;
+  cursor: pointer;
+  -webkit-transition: all 0.2s;
+  -moz-transition: all 0.2s;
+  -o-transition: all 0.2s;
+  transition: all 0.2s;
+}
+.modalCloseButton:hover {
+  border: 2px solid var(--text-white);
+}
+
+.connectWalletModal {
+  position: fixed;
+  display: flex;
+  flex-direction: column;
+  width: 61%;
+  max-width: 897px;
+  min-width: 300px;
+  height: fit-content;
+  background: var(--bg-color);
+  box-shadow: 0px 0px 4px rgba(255, 255, 255, 0.25);
+  border-radius: 24px;
+  padding: 30px;
+  justify-content: center;
+  align-items: center;
+  row-gap: 1em;
+}
+.walletConnectButton {
+  display: flex;
+  width: 100%;
```

Comment:
nit: you don't need to set `width:100%` in flexbox with no other elements

Reply:
Apparently you do? I think the image/text inside counts as an element, just defaults to fit-content.